lib/sycamore/sycamore/llms/prompts/default_prompts.py

```diff
@@ -169,6 +169,26 @@ def __init__(self, question: Optional[str], field: Optional[str], format: Option
             )
 
 
+class LLMFilterMessagesPrompt(OpenAIMessagesPromptBase):
+    def __init__(self, filter_question: Optional[str]):
+        super().__init__()
+
+        self.add_message(
+            "system",
+            ("You are a helpful classifier that generously filters database " "entries based on questions."),
```

Comment:
are the double quotes after database doing anything?

Reply:
oops nope